src/classes/particle.ts

```diff
@@ -332,32 +205,44 @@ export class Particle {
     //     img.src = url;
     // }
 
-    public grab() {
-        let container = this.container;
-        let options = container.options;
+    public grab(): void {
+        const container = this.container;
+        const options = container.options;
 
         if (options.interactivity.events.onhover.enable && container.interactivity.status === "mousemove") {
-            let dx_mouse = this.position.x - (container.interactivity.mouse.pos_x || 0);
-            let dy_mouse = this.position.y - (container.interactivity.mouse.pos_y || 0);
-            let dist_mouse = Math.sqrt(dx_mouse * dx_mouse + dy_mouse * dy_mouse);
-            /* draw a line between the cursor and the particle if the distance between them is under the config distance */
+            const dx_mouse = this.position.x - (container.interactivity.mouse.pos_x || 0);
+            const dy_mouse = this.position.y - (container.interactivity.mouse.pos_y || 0);
+            const dist_mouse = Math.sqrt(dx_mouse * dx_mouse + dy_mouse * dy_mouse);
+            /*
+               draw a line between the cursor and the particle
+               if the distance between them is under the config distance
+            */
             if (dist_mouse <= options.interactivity.modes.grab.distance) {
-                let opacity_line = options.interactivity.modes.grab.line_linked.opacity - (dist_mouse / (1 / options.interactivity.modes.grab.line_linked.opacity)) / options.interactivity.modes.grab.distance;
+                const opacity_line = options.interactivity.modes.grab.line_linked.opacity - (dist_mouse / (1 / options.interactivity.modes.grab.line_linked.opacity)) / options.interactivity.modes.grab.distance;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)